src/cmdclass.py

```diff
@@ -111,6 +111,10 @@ def run(self):
             # it with an updated value
             target_versionfile = os.path.join(self.build_lib,
                                               cfg.versionfile_build)
+            if not os.path.exists(target_versionfile):
+                print("Warning: %s does not exist, skipping version update" %
+                      target_versionfile)
```

Comment:
```suggestion
                print(f"Warning: {target_versionfile} does not exist, skipping version update. "
                      "This can happen if you are running build_ext without first running build_py.")
```